src/ZombieDriver.php

```diff
@@ -94,13 +94,14 @@ public function stop()
     public function reset()
     {
         $js = <<<JS
-browser.destroy();
-browser = null;
```

Comment:
That's interesting. Then how do I check, in Zombie case, that switch to main window actually occurred at all after `reset` method was called? Right now, when I attempt to find text on last opened page I end up with Fatal Error, because `browser.destroy();` closes all opened windows.

It sort of does switch back to last opened window, but I have no way to check if that really happens.